velox/docs/functions/spark/comparison.rst

```diff
@@ -0,0 +1,56 @@
+=====================================
+Comparison Functions
+=====================================
+
+.. function:: between(x, min, max) -> boolean
+
+    Returns true if x is within the specified [min, max] range
+    inclusive. Supports TINYINT, SMALLINT, INTEGER, BIGINT, DOUBLE,
+    REAL types. The types of all arguments must be the same.
+
+.. function:: equalto(x, y) -> boolean
```

Comment:
Is equalto a valid SparkSQL function?

Reply:
Just added "Function equalto corresponds to Spark's operator ``=`` and ``==``."